rts/lra/service-base/src/main/java/io/narayana/lra/LRAConstants.java

```diff
@@ -42,7 +42,14 @@
     public static final String RECOVERY_PARAM = "recoveryCount";
     public static final String HTTP_METHOD_NAME = "method"; // the name of the HTTP method used to invoke participants
 
-    public static final long PARTICIPANT_TIMEOUT = 2; // number of seconds to wait for requests
+    /**
+     * System property name which defines the connection read timeout which is used
+     * by LRA Coordinator and its friends when connecting remote participants and other coordinators.
+     * The value is in seconds. The default value is {@value LRAConstants#PARTICIPANT_DEFAULT_TIMEOUT_S}.
+     */
+    public static final String PARTICIPANT_TIMEOUT_PARAM = "lra.participation.timeout";
```

Comment:
Should we prefix these properties with `narayana` to avoid any potential conflicts with future LRA spec versions.

Also participants/compensators are referred to as participants or compensators so it would be more consistent to call this property `lra.participant.timeout`